packages/mobile/src/components/Chat/Chat.test.tsx

```diff
@@ -3903,7 +3903,7 @@ describe('Chat component', () => {
                     </View>
                   </View>
                 </View>
-              </View>
+              </div>
```

Comment:
this look suspicious

there are no divs in mobile. What is this doing here?

Reply:
That's super weird. It was supposed to be the opposite. Must have done something wrong when merging master